src/inputs/dropdown/query-search-dropdown/PQuerySearchDropdown.vue

```diff
@@ -167,7 +166,16 @@ export default defineComponent<QuerySearchDropdownProps>({
             onPaste,
             onDeleteAll,
             preTreatSelectedMenuItem,
-        } = useQuerySearch(props, { strict: true });
+        } = useQuerySearch(
+            {
+                value: props.value,
+                focused: props.focused,
+                valueHandlerMap: computed(() => props.valueHandlerMap),
+                keyItemSets: computed(() => props.keyItemSets),
+                visibleMenu: visibleMenuRef,
+            },
```

Comment:
How about this? 
```suggestion
            {
               ...props,
                visibleMenu: visibleMenuRef,
            },
```

Reply:
@yuda110 

```typescript
{
     ...props,
      visibleMenu: visibleMenuRef,
},
```

with this way, reactivity will be broken.


```typescript
{
     ...toRefs(props),
      visibleMenu: visibleMenuRef,
},
```

(I didn't check the code above works... )
Because `valueHandler`, `keyItemSets` must be reactive, must bridge props with reactive tools.